client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -121,6 +121,9 @@
     @Option(names = "--client-tags", paramLabel = "<tags>", description = "Client tags")
     public String clientTags;
 
+    @Option(names = "--routing-group", paramLabel = "<routingGroup>", description = "Routing group")
+    public String routingGroup;
```

Comment:
I think we can use `Optional<String>` here. I'm not sure why `traceToken` and a few others use nullable `String`, but that's probably legacy code that should be cleaned up at some point.